platform_cli/groups/release.py

```
@@ -374,6 +410,157 @@ def _build_deb_in_docker(
+        package_info: PackageInfo,
+        docker_image_name: str,
+        buildx_secrets: List[str],
+        package_dir: str,
+        ros_distro: str,
+        package: str,
+        docker_platforms: Optional[List[str]] = None,
+        use_registry: bool = False,
+    ) -> Optional[Manifest]:
+        """Build docker image using buildx with unified logic for both native and multiplatform builds"""
+        if not package_info.module_info:
+            raise Exception("Module info is required to build docker images")
```

Ruff (0.14.7)

Create your own exception

(TRY002)